cluster/manifests/wiz/connector/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+{{ if eq .Cluster.ConfigItems.wiz_enable_runtime_monitoring_daemonset "true"}}
+---
+# Source: wiz-kubernetes-integration/charts/wiz-kubernetes-connector/charts/wiz-broker/templates/wiz-broker-deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wiz-connector-agent
+  namespace: "wiz"
+  labels:
+    helm.sh/chart: wiz-broker-2.1.0
+    application: "wiz"
+    component: "connector"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: "wiz"
+      component: "connector"
+  template:
+    metadata:
+      labels:
+        helm.sh/chart: wiz-broker-2.1.0
+        application: "wiz"
+        component: "connector"
+    spec:
+      serviceAccountName: wiz-broker
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1000
+      volumes:
+        - name: connector-data
+          secret:
+            secretName: wiz-connector-connector
+            items:
+              - key: connectorData
+                path: data
+      containers:
+        - name: wiz-broker
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
+            runAsUser: 1000
+          image: "container-registry-test.zalando.net/secops-systems/wiz-broker:latest"
```

Comment:
`latest` tag will not work in production. So, we definitely need to fix this to the latest build version before merging. It's fine for testing.